src/UI/atoms/Text.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import styled, { css, FlattenSimpleInterpolation } from "styled-components";
+
+import fontSizes from "<styles>/variables/fontSizes";
+import fontFamilies from "<styles>/variables/fontFamilies";
+
+type FontSizesKeys = keyof typeof fontSizes;
+type FontFamilyKeys = keyof typeof fontFamilies;
+
+export interface TextProps {
+  children: JSX.Element | string;
+  fontSize?: typeof fontSizes[FontSizesKeys];
+  fontFamily: typeof fontFamilies[FontFamilyKeys];
+}
+
+function Text({ 
```

Comment:
```suggestion
const Text = ({ }) => (
   ....
)
```
This component contains only JSX